### ISSUE TITLE
204 No Content stubbed response not being handled

### ISSUE THREAD

**Issue by abraramin** [AlanCS/SystemTestingTools#1]
I am using `SystemTestingTools` `Version 1.3.6` . I used the recorder function to capture a 204 responce (as suggested in the project README). Then I intended to use this captured response as stub for component tests. This is what the response looks like:

```
METADATA
...

REQUEST
...

--!?@Divider: Any text BEFORE this line = comments, AFTER = response in Fiddler like format

HTTP/1.1 204 NoContent
Server:Microsoft-IIS/10.0
X-Powered-By:ASP.NET
Date:Tue, 05 May 2020 03:41:23 GMT
```
There is no response body (as expected). I am using `ResponseFactory.FromFiddlerLikeResponseFile("/path/to/stubfile")`

I get the following exception:
`System.ArgumentException : File is not in the right format, please consult https://github.com/AlanCS/SystemTestingTools/`

Modifying the captured response and using `null` as response content is my workaround for now.

**Comment by AlanCS** [comment]
theoretically fixed in https://www.nuget.org/packages/SystemTestingTools/1.3.7, please confirm

**Comment by abraramin** [comment]
I'm still getting the same error using 1.3.7 😕

**Comment by AlanCS** [comment]
Sorry it took so lon @abraramin, but I de-prioritized it as there as a workaround.
it should be fixed in version 1.3.10; let me know !

**Comment by AlanCS** [comment]
closing due to no further feedback
